spec/Appendix A -- Notation Conventions.md

```diff
@@ -216,3 +216,34 @@ Fibonacci(number):
 Note: Algorithms described in this document are written to be easy to
 understand. Implementers are encouraged to include equivalent but optimized
 implementations.
+
+## Data Collections
+
+This specification describes the semantic properties of data collections using
+types like "list", "set" and "map". These describe observable data collections
+such as the result of applying a grammar and the inputs and outputs of
+algorithms. They also describe unobservable data collections such as temporary
+data internal to an algorithm. Each data collection type defines the operations
+available, and whether values are unique or ordered.
+
+**List**
+
+:: The term _list_ describes a sequence of values which may not be unique. A
+list is ordered unless explicitly stated otherwise (as an "unordered list"). For
+clarity the term "ordered list" may be used when an order is semantically
+important.
+
+**Set**
+
+:: The term _set_ describes a unique collection of values, where each value is
+considered a "member" of that set. A set is unordered unless explicitly stated
```

Comment:
Is it the collection that is unique or its content? Having "unique collection" feels like the former to me (but maybe it's a thing?).


Tentative disambiguation:
```suggestion
:: The term _set_ describes a unique collection of values. Each value is
considered a "member" of that set and must be unique in the set. A set is unordered unless explicitly stated
```